### ISSUE TITLE
SendInBlue integration doesn't show cutom contact fields to map against

### ISSUE THREAD

**Issue by russback** [verbb/formie#346]
**Description**
When connecting a form, you only get the core fields such as email and name fields. None of the custom filds you create are available in the list to map to.

This is an important one to capture fields such as consent for email, SMS, etc, and attribution such as campaign data capturede from UTM variables.

**Steps to reproduce**
1. In SiB, add a custom contact attribute ate https://my.sendinblue.com/lists/add-attributes
2. Create a form and see that the new attribute is not available in the Field Mappings

### Form settings
- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: Yes
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.17
- Craft version: 3.6.5.1
- Multi-site: Yes

**Comment by engram-design** [comment]
I remember looking at this at the time, not knowing how to fetch them via the API.

Added for the next release. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.17",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.3.18](https://github.com/verbb/formie/releases/tag/1.3.18)
